src/user_templates_api/templates/jupyter_lab/templates/hra_api_tutorial/metadata.json

```diff
@@ -0,0 +1,19 @@
+{
+  "title": "Human Reference Atlas (HRA) API Tutorial",
+  "description": "This notebook shows users how to instantiate and use the HuBMAP HRA APIs.",
+  "tags": [
+    "hra",
+    "api"
+  ],
+  "is_multi_dataset_template": true,
+  "template_format": "jinja",
+  "examples": [
+    {
+      "title": "Working with the HRA API examples",
+      "description": "Example metadata and available files for four datasets.",
```

Comment:
Same comment as for the other template, please add a description about this template

Reply:
I removed the example datasets. Can this whole examples section be deleted then? I'm confused what goes there if there are not example datasets.